### DISCUSSION TITLE
How to set radio box form field value?

### DISCUSSION THREAD

**Discussion by saadgit** [py-pdf/pypdf#1283] [Q&A]
Sample pdf:  https://royalegroupnyc.com/wp-content/uploads/seating_areas/sample_pdf.pd,

Upon reading the form fields like `reader.get_fields()`, I get the following fields and here Group6 is the radio box. 

<img width="1161" alt="Screen Shot 2022-08-26 at 12 01 23 PM" src="https://user-images.githubusercontent.com/3226154/186946319-0455c7cc-9115-4bb8-a088-fda4a2c0e856.png">

Question is how to update this form field value?

**Comment by MartinThoma** [comment]
Did you find a solution? I would suggest to use https://github.com/py-pdf/PyPDF2/discussions/1360 to discuss this problem. Maybe somebody has an idea :-) :crossed_fingers:

**Comment by fernandesjeevan** [comment]
@saadgit I know its late, but did you find a solution for this problem, thanks in advance
